tests/scripts/script5.py

```diff
@@ -0,0 +1,6 @@
+
+try:
+    import odoo.addons.addon1  # noqa
+except ImportError:
+    #  Odoo < 10.0
+    import openerp.addons.addon1  # noqa
```

Comment:
Hm, this is not correct as the absence of addon1 in addons-path would raise the wrong error. Ideally there should be a test that checks script5 fails if addon1 is not in addons-path.

And this script should be

```python
# noqa
try:
    import odoo
except ImportError:
    #  Odoo < 10.0
    try:
        import openerp
    except ImportError:
        # odoo not found
        raise
    else:
        import openerp.addons.addon1
else:
    import odoo.addons.addon1
```

Reply:
Oh thanks!